blog/2021-05-05-the-curse-of-the-a-word.md

```diff
@@ -0,0 +1,102 @@
+---
+tags:
+  - v1
+  - Temporal
+  - microservice-orchestration
+  - microservices
+  - orleans
+posted_on_: 2021-05-05T07:00:00Z
+slug: sergey-the-curse-of-the-a-word
+title: 'The Curse of the A-word'
+author: Sergey Bykov
+author_title: Engineering
+author_image_url: https://avatars2.githubusercontent.com/u/8248806?s=460&v=4
+image: https://avatars2.githubusercontent.com/u/8248806?s=460&v=4
+release_version: V1.9.1
+---
+
+<!--truncate-->
+
+![Actors](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/rlw56fpp0hjgd73xmq8h.png)
+
+I wanted to write this post ever since I saw David Fowler's tweet and the discussion it triggered.
+![David Fowler](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/qt8irq2n337p5tsf7u4i.png)
+While it may have taken me quite some time to get around to answering it, 2020 wasn't an ordinary year by any measure.
```

Comment:
- your opening para is what people use to decide if they will read the rest
- i dont know who david fowler is. most people outside .net may not know who he is.
- using screenshot of tweet is inaccessible for screen readers
- mentioning 2020 is a reason, yes, but is it important enough to put in the first para? is there some better hook to get interest?

```suggestion

**Why aren’t actor frameworks more popular?**

I've wanted to write this post ever since [David Fowler's tweet](https://twitter.com/davidfowl/status/1189031080076009472?lang=en) and the discussion it triggered.

In short: Naming it "actors" caused a minefield of conflations, and both Microsoft Orleans and Cloudflare Workers have avoided them by actively avoiding the word. You should too.
```

this more directly answers the question posed by the blog title and invites the reader to read on.

just a suggestion